### DISCUSSION TITLE
Rename Open chats to Find chats to reduce the keystrokes

### DISCUSSION THREAD

**Discussion by LintaoAmons** [olimorris/codecompanion.nvim#266] [General]
## Description

When type `Open` in the `CodeCompanionAction` telescope picker. Two of the options will left and user need to use extra keystroke to select the one they need then enter.

<img width="950" alt="image" src="https://github.com/user-attachments/assets/7bee2049-bd3b-4850-8fc8-ad8bc8741a76">

But simply change the name of the command to `Find` could filter out all other options

<img width="944" alt="image" src="https://github.com/user-attachments/assets/9213830d-7887-48f6-81f5-ac4f93079cf9">

And simplify the flow.

**Comment by S1M0N38** [comment]
Another option is to change description of `Chat` to "Create a new chat buffer to converse with an LLM"

**Reply by olimorris** [reply]
Great shout. I've gone for this in 4bec50da26b411e52accdabd358e7c00ff94d2d3
